src/AWS/Orleans.Streaming.SQS/Storage/SQSStorage.cs

```diff
@@ -230,10 +313,71 @@ await sqsClient.DeleteMessageAsync(
             }
         }
 
+        public async Task DeleteMessages(IEnumerable<SQSMessage> messages)
+        {
+            try
+            {
+                foreach (var message in messages)
+                {
+                    ValidateMessageForDeletion(message);
+                }
+
+                foreach (var batch in messages.Chunk(MAX_NUMBER_OF_MESSAGE_TO_PEEK))
+                {
+                    var deleteRequest = new DeleteMessageBatchRequest
+                    {
+                        QueueUrl = queueUrl,
+                        Entries = batch
+                            .Select((m, i) =>
+                                new DeleteMessageBatchRequestEntry(i.ToString(), m.ReceiptHandle))
+                            .ToList()
+                    };
+
+                    var result = await sqsClient.DeleteMessageBatchAsync(deleteRequest);
+                    foreach (var failed in result.Failed)
+                    {
+                        Logger.LogWarning("Failed to delete message {MessageId} from SQS queue {QueueName}. Error code: {ErrorCode}. Error message: {ErrorMessage}",
```

Comment:
Minor: can you update to use a [message template](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#log-message-template)